change/@uifabric-experiments-2020-01-15-16-19-24-users-v-jajach-fixed-issue-11714.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Fixed #11714",
```

Comment:
pls make the comment more detailed, suggestion:
TextField: fix focused bottom border mis-alignment when errorMessage is set